### ISSUE TITLE
Only list canonical URLs for user profiles in sitemap

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-server#17071]
In https://github.com/mozilla/addons-server/issues/16960, some user profile URLs have been added but there are too many URLs because of pagination (I think). User profiles might display two sets of add-ons: one for extensions, and one for themes. There are two query parameters for that: `page_e` and `page_t`.

Each page has a canonical URL on AMO, and for user profile pages, there are a few rules (cf. https://github.com/mozilla/addons-frontend/issues/6527). Here is the code responsible for generating "correct" (SEO-wise) URLs: https://github.com/mozilla/addons-frontend/blob/09c10da237835a6f1a9d829a053d3e01822456df/src/amo/components/UserProfileHead/index.js#L27-L64

In other words:

- we don't want to return a query string when values are '1' (I think this is already covered)
- when both parameters are provided, we take the one with the highest value
- when both values are equal, we must respect the order in the query string

In other words, addons-frontend generates canonical URLs with up to one query parameter for those user profile pages.

**Comment by AlexandraMoga** [comment]
I've verified some of the user profile pages added to the sitemap, insisting on those that include pagination:

User example from the [sitemap](https://addons-dev.allizom.org/sitemap.xml?section=users&app_name=firefox&debug): https://addons-dev.allizom.org/en-US/firefox/user/10640485/ => has 22 pages of extensions and 12 pages with themes; these are all listed subsequently in the sitemap.

I think this was the expected behavior.
